api/models.py

```diff
@@ -95,7 +94,8 @@ class User(Base):
     """Models Users."""
 
     __tablename__ = 'users'
-    email = db.Column(db.String)
+    name= db.Column(db.String, nullable=False, unique=True)
```

Comment:
It's possible that there might be more than one Andelan with the same name.

Reply:
Good point. I have removed the `unique` value